graphics/macgui/macmenu.cpp

```diff
@@ -1106,6 +1103,33 @@ bool MacMenu::draw(ManagedSurface *g, bool forceRedraw) {
 	return true;
 }
 
+bool MacMenu::drawCustom(ManagedSurface* g, int x, int y, bool forceRedraw) {
```

Comment:
No, please overload `draw()` method instead of this ugly `drawCustom()`

Also, this line does not conform to our code formatting conventions